packages/extensions/openapi-to-typespec/test/arm-alertsmanagement/tsp-output/Alert.tsp

```diff
@@ -0,0 +1,130 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Foundations;
+using TypeSpec.Http;
+
+namespace Azure.ResourceManager.AlertsManagement;
+@doc("An alert created in alert management service.")
+@subscriptionResource
+model Alert is ProxyResource<AlertProperties> {
```

Comment:
What is the best practice for models?
Can this model also be moved into models.tsp?

Reply:
resource's model will be placed in the corresponding resource's TSP file.